index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My_Portfolio</title>
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+      integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&family=Poppins:ital@0;1&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+      integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <main>
+      <header>
+        <div class="container-nav">
+          <a href="index.html" class="logo-title">Marlyn Mayienga</a>
+          <nav class="nav-bar">
+            <ul>
+              <li class="portfolio">
+                <a href="#portfolio">Portfolio </a>
+              </li>
+              <li>
+                <a href="#">About </a>
+              </li>
+              <li>
+                <a href="#contact">Contact </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+        <div class="hamburger-btn">
+            <i class="fas fa-bars"></i>
```

Comment:
- [ ] It would be better if you place hamburger menu icon in the same place as in the Figma design 
![image](https://user-images.githubusercontent.com/61790587/176030736-0099bb8c-c2b9-41f2-9d3f-aa478e73b41b.png)